container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_collector_functions/implementations/fluentbit/consts.go

```diff
@@ -62,5 +56,10 @@ const (
 	matchAllRegex          = "*"
 	jsonLineFormat         = "json"
 	unlimitedOutputRetry   = "no_limits"
+	lokiOutputTypeName     = "loki"
```

Comment:
I'm confused by this. do we still have loki around?

Reply:
going to remove loki impl